workflow-core/src/main/java/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -85,7 +85,13 @@ abstract class StatefulWorkflow<
     out RenderingT
     > : Workflow<PropsT, OutputT, RenderingT> {
 
-  inner class RenderContext internal constructor(
+  open inner class RenderContext internal constructor(
+    baseContext: BaseRenderContext<PropsT, StateT, OutputT>
+  ) : BaseRenderContext<@UnsafeVariance PropsT, StateT, @UnsafeVariance OutputT> by baseContext
+
+  inner class RenderContextDeluxe internal constructor(
```

Comment:
In similar sketches I've done, since this change is largely to reduce boilerplate, I just call this type `Ctx` so it doesn't take up as much visual space, or as long to type if people are typing these by hand.